src/python/tensorflow_cloud/core/tests/unit/deploy_test.py

```diff
@@ -184,9 +184,9 @@ def test_request_dict_without_user_args(self, MockDiscovery):
             },
         )
 
+    @patch("tensorflow_cloud.core.validate.VERSION", "2.1.0")
```

Comment:
This shouldn't be required here, can you check?

Reply:
you are correct, updating accordingly.